docs/api-keys-and-rate-limits.md

```diff
@@ -0,0 +1,22 @@
+# API keys and rate limits
+
+## Obtain an API key
+
+To use Mapzen Terrain Tiles, you should first obtain a free developer API key. Sign in at https://mapzen.com/developers to create and manage your API keys.
+
+1. Go to https://mapzen.com/developers.
+2. Sign in with your GitHub account. If you have not done this before, you need to agree to the terms first.
+3. Create a new key for Mapzen Terrain Tiles, and optionally, give it a name so you can remember the purpose of the project.
+4. Copy the key into your code.
+
+## Rate limits
```

Comment:
We need to update https://mapzen.com/documentation/overview/#mapping-products to restate these limits?